### PR TITLE
Bug 1088450 - improve Bluetooth transfer messages, r=tim

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1abd40db34fe27806d1343106d4965e298704513

File: apps/system/js/bluetooth_transfer.js [hunk 1]
```diff
@@ -424,26 +426,25 @@ var BluetoothTransfer = {
       return;
     }
 
-    finishedTask.removeEventListener('click',
```
Comment (gasolin) [inline apps/system/js/bluetooth_transfer.js]: that was wrong because bind function return a new object

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d64b8019c1039311c56c5be2d9bf6bf33fbfe79a

File: apps/system/js/bluetooth_transfer.js (hunk 1 above)
Comment (timdream) [inline apps/system/js/bluetooth_transfer.js]: ... and memory leaks. Thank you.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3e386065b7bd98e888451776bd02c21b201ad44a

**Pull request merged**